packages/react/src/components/AnimatedHeader/components/AnimatedHeader/AnimatedHeader.tsx

```diff
@@ -26,9 +26,22 @@ import TasksController, {
 import WorkspaceSelector, {
   WorkspaceSelectorProps,
 } from '../WorkspaceSelector/WorkspaceSelector';
+import HeaderTitle from '../HeaderTitle/HeaderTitle';
 
 /** Animated Header */
 
+export interface AriaLabels {
+  welcome?: string;
+  description?: string;
+  collapseButton?: string;
+  expandButton?: string;
+  tilesContainer?: string;
+  tiles?: { [tileId: string]: string };
+  tasksDropdown?: string;
+  workspaceDropdown?: string;
+  [key: string]: any;
```

Comment:
Why do we need that? I believe we won't use anywhere the arbitrary key values anyway

Reply:
There is no need I will remove.